docs/source/federation/implementing.md

```diff
@@ -97,38 +112,78 @@ server.listen(4001).then(({ url }) => {
     console.log(`🚀 Server ready at ${url}`);
 });
 ```
-> Note: we're now providing a `schema` to the `ApolloServer` constructor, rather than `typeDefs` and `resolvers`.
 
-Now the service is running as a federated service and is ready to be composed into an overall federated graph!
+The server is now ready to be added to a federated data graph!
+
+Here's the complete code sample for our federation-ready server:
```

Comment:
Is it "complete" if `fetchUserById` is not defined? I don't want people to copy this and be surprised it doesn't compile :P

Reply:
Good clarification